AutoGear.lua

```diff
@@ -2853,22 +2909,17 @@ function AutoGearReadItemInfo(inventoryID, lootRollID, container, slot, questRew
 	if (info.BlueSockets == 0) then info.BlueSockets = nil end
 	if (info.MetaSockets == 0) then info.MetaSockets = nil end
 
-	local classId = 0
-	if info.id then
-		classId = select(12, GetItemInfo(info.id))
-	end
-
 	if (info.Slot) then
 		info.shouldShowScoreInTooltip = 1
 	end
 	if (not cannotUse and (info.Slot or info.isMount)) then
 		info.Usable = 1
-	elseif classId == 15 then
+	elseif itemClassId == 15 then
 		cannotUse = 1
-		reason = "(classId = " .. classId .. ", it is not usable)"
+		reason = "(classId = " .. itemClassId .. ", it is not usable)"
```

Comment:
The `classId` part of this string doesn't match the new variable name.  Please update that to `itemClassId`.